apps/condo/domains/miniapp/schema/B2BAppNewsSharingConfig.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createschema miniapp.B2BAppNewsSharingConfig 'publishUrl:Text; previewUrl:Text; getRecipientsUrl:Text;'`
+ */
+
+const { Url, Text } = require('@keystonejs/fields')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2BApp')
+
+
+/**
+ * News Sharing B2BApp
+ * 
+ * News Sharing B2BApp allow b2b users to share their NewsItem to external source like Telegram or Whatsapp from /news page
+ *
+ *                         [ whatsapp-sharing-miniapp ] -> [ whatsapp ]
+ * [ condo /news page ] ->              ...                    ...
+ *                         [ telegram-sharing-miniapp ] -> [ telegram ]
+ *
+ * To create miniapp that can be embedded to /news page developer should provide API and information defined here
+ * 
+ * @type {GQLListSchema}
+ */
+const B2BAppNewsSharingConfig = new GQLListSchema('B2BAppNewsSharingConfig', {
+
+    schemaDoc: 'News Sharing B2BApp allow b2b users to share their NewsItem to external source (like Telegram) from /news page',
```

Comment:
Can a single miniapp implement multiple features or should some validations be added if not?

Reply:
Hey, thanks for the comment

Miniapp can implement different features, that be `NewsSharing` and `PropertyMapExtensions`. Miniapp cannot implement two `NewsSharings`

I do not see what validations should be added, want to keep is simple, really